.eslintrc.js

```diff
@@ -3,6 +3,7 @@ const finalRules = {
   'react/jsx-filename-extension': [0],
   'react/no-render-return-value': [0],
   'react/no-array-index-key': [0],
+  'react/destructuring-assignment': [0],
```

Comment:
the chain of upgrades forced me to upgrade eslint config airbnb which added new rule. So i disabled the new rule for now.